lib/ops/opsservice/configure.go

```diff
@@ -323,9 +323,11 @@ func (s *site) configurePackages(ctx *operationContext, req ops.ConfigurePackage
 		if err != nil {
 			return trace.Wrap(err)
 		}
-		if s.cloudProviderName() != "" {
-			clusterConfig.SetCloudProvider(s.cloudProviderName())
-		}
+	} else {
+		clusterConfig = clusterconfig.NewEmpty()
```

Comment:
Nit: Instead of `else` block you could've probably just initialize empty clusterConfig above.

```go
clusterConfig := clusterconfig.NewEmpty()
if len(req.Config) != 0 {
  clusterConfig, err = clusterconfig.Unmarshal(req.Config)
  ...
}
```